test/AssemblySet_basic_test.py

```diff
@@ -1,54 +1,41 @@
 # -*- coding: utf-8 -*-
 import os
 import shutil
-import unittest
 from pprint import pprint
 from test import TEST_BASE_DIR
-from test.conftest import INFO_LENGTH, WS_NAME, test_config
+from test.base_class import BaseTestClass
+from test.conftest import INFO_LENGTH
 
-from installed_clients.AssemblyUtilClient import AssemblyUtil
 
-
-class SetAPITest(unittest.TestCase):
+class AssemblySetAPITest(BaseTestClass):
     @classmethod
-    def setUpClass(cls):
-        props = test_config()
-        for prop in ["cfg", "ctx", "serviceImpl", "wsClient", "wsName", "wsURL"]:
-            setattr(cls, prop, props[prop])
+    def prepare_data(cls: BaseTestClass) -> None:
+        """Set up fixtures for the class.
 
+        :param cls: class object
+        :type cls: BaseTestClass
+        """
         # copy test file to scratch area
         fna_filename = "seq.fna"
         fna_path = os.path.join(cls.cfg["scratch"], fna_filename)
         shutil.copy(os.path.join(TEST_BASE_DIR, "data", fna_filename), fna_path)
 
-        au = AssemblyUtil(os.environ["SDK_CALLBACK_URL"])
-        cls.assembly1ref = au.save_assembly_from_fasta(
+        cls.assembly1ref = cls.au.save_assembly_from_fasta(
             {
                 "file": {"path": fna_path},
-                "workspace_name": WS_NAME,
```

Comment:
got a bit ahead of myself in the prev PR so just reverting this change for now (applies throughout the PR)